Emzzy241.md

```diff
@@ -0,0 +1,73 @@
+# Emmanuel Mojiboye
+
+📍 Ifako-Ijaiye, Lagos-Nigeria  
+📞 +2349012857304  
```

Comment:
I would suggest do not include phone number in public repo unless you are okay with some spam